From 1aad31c1cf6d8c5dadfdc4145594deef1a6fc0f7 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Motiejus=20Jak=C5=A1tys?= <motiejus.jakstys@spilgames.com>
Date: Fri, 12 Apr 2013 18:28:10 +0200
Subject: [PATCH 7/9] Fix test.cpp compile error

On gcc 4.7 this error happens:

    test.cpp: In function 'int main(int, char**)':
    test.cpp:28:13: error: 'close' was not declared in this scope
---
 tests/cpp/test.cpp |    1 +
 1 file changed, 1 insertion(+)

diff --git a/tests/cpp/test.cpp b/tests/cpp/test.cpp
index 9332be7..7a7a49c 100644
--- a/tests/cpp/test.cpp
+++ b/tests/cpp/test.cpp
@@ -2,6 +2,7 @@
 #include <sys/stat.h>
 #include <fcntl.h>
 #include <stdlib.h>
+#include <unistd.h>
 
 #include <iostream>
 #include <google/protobuf/text_format.h>
-- 
1.7.9.5

